sdk/python/examples/jobset.py

```diff
@@ -0,0 +1,43 @@
+import jobset
+from jobset.models.jobset_v1alpha2_job_set import JobsetV1alpha2JobSet
+from kubernetes.client.models.v1_job_template_spec import V1JobTemplateSpec
+
+jobset_example = jobset.models.jobset_v1alpha2_job_set.JobsetV1alpha2JobSet(
+    api_version="0",
+    kind="0",
+    metadata=None,
+    spec=jobset.models.jobset_v1alpha2_job_set_spec.JobsetV1alpha2JobSetSpec(
+        failure_policy=jobset.models.jobset_v1alpha2_failure_policy.JobsetV1alpha2FailurePolicy(
+            max_restarts=56,
+        ),
+        replicated_jobs=[
+            jobset.models.jobset_v1alpha2_replicated_job.JobsetV1alpha2ReplicatedJob(
+                name="0",
+                network=jobset.models.jobset_v1alpha2_network.JobsetV1alpha2Network(
+                    enable_dns_hostnames=True,
+                ),
+                replicas=56,
+                template=V1JobTemplateSpec(),
+            )
+        ],
+        success_policy=jobset.models.jobset_v1alpha2_success_policy.JobsetV1alpha2SuccessPolicy(
+            operator="0",
```

Comment:
Success policy operator should be "Any" or "All" (see [here](https://github.com/kubernetes-sigs/jobset/blob/main/api/jobset/v1alpha2/jobset_types.go#L155-L158)). However, as mentioned in a previous comment, we can just let the success policy defaulted instead.

Reply:
done.